### DISCUSSION TITLE
Turned on raptor and parse failed

### DISCUSSION THREAD

**Discussion by ronchan086-fd** [infiniflow/ragflow#2466] [Q&A]
Hi ,

When I added a document into a knowledgebase and have raptor option turned on, it failed with the following message. Any suggestion please.

Page(37~47): OCR is running...
Page(37~47): OCR finished.
Page(37~47): Layout analysis finished
Page(37~47): Table analysis finished
Page(37~47): Text merging finished.
Page(37~47): Finished slicing files(15). Start to embedding the content.
Page(37~47): Finished embedding(0.30)! Start to build index!
Page(37~47): Done!
Start to do RAPTOR (Recursive Abstractive Processing For Tree-Organized Retrieval).
Task has been received.
[ERROR]'ConnectionError' object has no attribute 'f_lineno'

Thanks in advance.

Ron

**Comment by KevinHuSh** [comment]
RAPTOR need to call LLM. It seems failure happened to call LLM.
Is chat functioning normally using that LLM?

**Reply by aamir-gmail** [reply]
Happened to me as well.I think the reason it failed because the default model for Raptor  is Qwen, any ideas on how to change this to ollama models

**Reply by KevinHuSh** [reply]
settings > model provider > system model settings

**Reply by aamir-gmail** [reply]
Thank you that worked.
